src/style.css

```diff
@@ -0,0 +1,128 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #eef;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 0 17px;
+
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.card__product-image {
+  padding: 32px 3px 0;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__product-name {
+  width: 166px;
+  height: 36px;
+
+  margin: 0 0 4px;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__product-code {
+  width: 98px;
+  height: 14px;
+
+  margin-bottom: 16px;
+
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__review {
+  display: flex;
+  align-items: center;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.stars {
+  display: flex;
+  height: 16px;
+}
+
+.star {
+  background: url(images/star.svg) center no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .star:nth-child(-n + 1),
+.stars--2 .star:nth-child(-n + 2),
+.stars--3 .star:nth-child(-n + 3),
+.stars--4 .star:nth-child(-n + 4),
+.stars--5 .star:nth-child(-n + 5) {
+  background: url(images/star-active.svg) center no-repeat;
+}
+
```

Comment:
You can make this section a separate file for easy reuse with @import

Reply:
furthermore — you have to